app/src/main/res/layout/state_fragment.xml

```diff
@@ -15,20 +15,31 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
     <LinearLayout
+      android:id="@+id/linearLayout"
```

Comment:
Can we name this ID based on what it contains? Also, prefer IDs to follow snake_case.

Consider: ``state_content_container``

Reply:
corrected.